interface/weno/confirm.php

```diff
@@ -0,0 +1,269 @@
+<link rel="stylesheet" href="../../library/js/jquery-ui.min.css" type="text/css" />
+
+<script type="text/javascript" charset="utf-8">
+function validate(){
+	var pharmacy = document.getElementById('pharm');
+	if(text.value.length == 0){
+		alert("Must select a pharmacy first");
+		return false;
+	}
+}
+
+
+</script>
+<style>
+footer {
+    padding: 0.5em;
+    font-size: 0.5em;
+
+    clear: left;
+    text-align: center;
+    top: 200px;
+}
+</style>
+</head>
+
+<body class="body_top">
+<h1>Prescription Transmit Review</h1>
+<table>
+<th width="200px">Drug</th>
+<th width="100px">Quantity</th>
+
+<?php
+//List drugs to be sent 
+
+  $drug = array(); //list of records that need to updated with pharmacy information
+while($list = sqlFetchArray($send)){
+    //print "<tr align='center'><td>".$list['drug'] . " </td><td> " . $list['dosage'] . " </td><td> " . $list['quantity'] . "</td></tr>";
+	print "<tr align='center'><td>".$list['drug'] . " </td><td> " . $list['quantity'] . "</td></tr>";
+	$drug[] = $list['id'];
+}
+
+
+?>
+</table>
+<?php if(empty($drug)){
+	echo "<br> <strong> <font color='red'>No prescriptions selected. </strong></font>";
+	exit;
+}
+?>
+<div id="fields">
+<h3>Select Pharmacy</h3>
+	    Patient Default <br>
+	    <input type = 'radio' name = "pharmacy" id = 'patientPharmacy' value="<?php print $patientPharmacy['pharmacy_id'] ?>" checked="checked">
+	    <?php if(!$patientPharmacy['name']){
+                   print "<b>Please set pharmacy in patient's chart!</b><br> <br>";
+               }else{
+	    	      print $patientPharmacy['name']; 
+               } 
+	    	      ?><br> <br>
+	    	      
+        Mail Order <br>
+        <input type = 'radio' name = 'pharmacy' id = 'mailOrder' value = "<?php print $mailOrder['id'] ?>">CCS Medical 	14255 49th Street, North, Clearwater, FL 33762<br> 
+	    <!-- removed from site but has future plans. 
+		<input type='text' size='10' name='city' id="city" value='' placeholder='Enter City First' title="type all or three letters of a city name">
+		<input type='text' size='30' name='address' id='address' value='' placeholder='Enter 3 #s or Letters of the Address' title="when searching by street name only put in the first three letters of the name"><br>
+		<input type='text' size='70' name='pharmacy_id' id="pharm" value='' class='pharmacy' placeholder='Enter City First Then Type Pharmacy' >
+		-->
+
+  <div id="confirm">
+  <br><br>
+      <input type='submit' id='confirm_btn' value='Aprove Order' >
+  </div>
+ 
+  <div id="transmit">
+      <input type='submit' id='order' value='Transmit Order' >
+  </div> 
+  <div id="success"></div>  
+</div>
+<script type="text/javascript" src="../../library/js/jquery-1.9.1.min.js"></script>
+<script type="text/javascript" src="../../library/js/jquery-1.6.4.min.js"></script>
+<script type="text/javascript" src="../../library/js/jquery-ui.min.js"></script>
+<script type="text/javascript" charset="utf-8" src="../../library/js/prism.js" ></script>
```

Comment:
Note that the library/js/jquery-ui.min.js does not exist, so not sure what this is doing . Because of this I can't figure out the version. If I knew the version of this, then I could point you to where it is in public/assets .